keyboards/iris/keymaps/wanleg/keymap.c

```diff
@@ -0,0 +1,76 @@
+
+#include QMK_KEYBOARD_H
+#include "wanleg.h"
+
+
+#define LAYOUT_iris_base( \
```

Comment:
:D

Reply:
what can i say? you're the best person to steal code from!